src/components/ImageSearchResultTable.tsx

```diff
@@ -58,30 +98,45 @@ export const columns: ColumnDef<SearchableImageWithGameCard>[] = [
   {
     accessorKey: "tags",
     header: () => "Tags",
+    // cell: ({ row }) => {
+    //   const tags: string[] = row.getValue("tags");
+    //   // return tags.join(", ");
```

Comment:
looks like some duplicate code from #63  ?

Reply:
Sorry... those can be removed.  Those were the original references in case I wanted to roll them back without git actions to undo.

And the import of classNames is required to dynamically compile my own class handling for the pills.